packages/remediations/src/RemediationButton.js

```diff
@@ -38,7 +38,7 @@ class RemediationButton extends React.Component {
 
     onClick = () => {
         Promise.resolve(this.props.dataProvider())
-        .then(data => this.state.remediations.openWizard(data))
+        .then(data => this.state.remediations.openWizard({ ...data, onRemediationCreated: this.props.onRemediationCreated }))
         .then(result => result && this.props.onRemediationCreated(result));
```

Comment:
We don't really need this line anymore, right?

Reply:
Yep. I think this logic can be simplified in new wizard